src/lap_stats/lap_stats.c

```diff
@@ -446,6 +444,8 @@ static void lapstats_location_updated(const GpsSnapshot *gps_snapshot)
     const GeoPoint *gp = &gps_snapshot->sample.point;
     const float target_radius = degrees_to_meters(config->TrackConfigs.radius);
 
+    update_distance(gps_snapshot);
+
```

Comment:
feedback is it should start calculating distance even if we're not in a lap - but resets once we start the beginning of a lap.